presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -149,6 +149,7 @@
     public static final String PARTITIONING_PRECISION_STRATEGY = "partitioning_precision_strategy";
     public static final String EXPERIMENTAL_FUNCTIONS_ENABLED = "experimental_functions_enabled";
     public static final String USE_LEGACY_SCHEDULER = "use_legacy_scheduler";
+    public static final String BLOCK_LARGE_BYTECODE_GENERATION = "block_large_bytecode_generation";
```

Comment:
Use disallow / disable? Or reverse the logic and name it allow...?

Reply:
Hmm - it's not stopping all of them. We still allow the asm compiler to fail fast for smaller queries still generating large bytecode. This is just weird flag.

Reply:
Actually just changed it to disallow.